src/box/relay_stream.h

```diff
@@ -0,0 +1,66 @@
+#pragma once
```

Comment:
Having it in a separate file made sense when it was EE plugin included into CE via `#include stream.cc` or something. But now perhaps it isn't so needed? And might even be reused someday for appliers? We coincidentally already have `box/xrow_io.h/c`. Do you think this struct would fit there?

I didn't think of this before, but as wrote above - I now think we might reuse it for appliers if eventually will make the appliers also do multiple tasks in different fibers. And that means the struct could be called `coio_funnel` or `coio_crater` - like "воронка": multiple fibers can write safely, and only one fiber is flushing into an `iostream`. The flush-size could be stored as a member and set once on the object creation.

What do you think of that?

Reply:
I think it fits good in `xrow_io.h/.c` now, indeed. Moved the stream there.

Speaking of the name - I'd settle with a more general `xrow_stream`. `xrow_io_stream`, maybe.

TBH I don't see how this could be reused by appliers: they mostly read, not write. And the only things they write are occasional heartbeats.

Reply:
For now I settled with `struct xrow_stream`.